### ISSUE TITLE
Make it possible to follow the book offline

### ISSUE THREAD

**Issue by gyzerok** [LukeMathWalker/zero-to-production#107]
Hello Luca and thank you for such an amazing book!

Currently if I'd decide to follow the book offline (let's say during the flight) I'd quite soon will be blocked by the fact, that I can't install some dependency or cargo extension. Of course I would be able to continue reading, but coding would be out of scope.

I was wondering if the book would benefit from 1 additional step which would combine all the downloads you need, which can be ran before following along and thus stored in cache?

**Comment by LukeMathWalker** [comment]
The easiest way to achieve an offline reading experience is to `git clone` the code repository and then run:
- `cargo check`;
- `cargo build`
- initialise the DB using the provided script (this gets you to install `sqlx-cli` + the Docker image for Postgres). 
Then you should be able to run everything offline using the `--offline` flag.  
The majority of the cargo extensions we end up using (e.g. `cargo-edit` / `cargo-expand`) are optional and do not block you from moving forward.

Unfortunately other parts of the book are less suited to offline reading - e.g. everything related to deployments/CI/usage of Digital Ocean.

**Comment by gyzerok** [comment]
I see, thanks!
